snuba/web/rpc/v1/resolvers/R_eap_items/storage_routing/routing_strategies/linear_bytes_scanned_storage_routing.py

```diff
@@ -307,6 +297,18 @@ def _output_metrics(self, routing_context: RoutingContext) -> None:
                 routing_context,
                 self.metrics.timing,
                 f"time_to_run_query_in_target_tier_{target_tier}",
-                self._get_query_duration_ms(routing_context.query_result),
+                query_duration_ms,
                 tags={"tier": str(target_tier)},
             )
+
+            if (
+                target_tier != Tier.TIER_1
+                and (self._get_time_budget_ms() - query_duration_ms)
+                / self._get_time_budget_ms()
+                >= 0.2
+            ):
+                self.metrics.increment(
+                    "query_should_run_on_higher_accuracy_tier",
+                    1,
+                    {"tier": str(target_tier)},
+                )
```

Comment:
rather than having these different metrics for the way the model fails, instead, write one metric call it `routing_mistake` or something, and tag it with the reason why it's an error. That way, we can easily see at a glance how many mispredictions we make and drill down further if needed